deployer/cilogon_app.py

```diff
@@ -148,25 +152,27 @@ def delete(self, id):
         Args:
            id (str): Id of the client to delete
 
-        Returns 200 OK or raises an error if the delete request returned anything other than 200.
+        Returns status code if response.ok or raises an error if the delete request returned anything not in the range 200-299.
```

Comment:
It doesn't raise an error in the python sense, but returns None and prints information about it.

My expectation is that we would somewhere after printing an error message do `sys.exit(1)` to make the exit with a non-0 status code.

This it not critical, and if would require systematic updates, then let's not burden this PR with that.

Reply:
Good point! I've updated the comment to match what's actually happening. Before this PR, it used to do `raise_for_status()`, but I removed it, because it just cluttered the screen with an unuseful error trace, when what we want really is the reason + status code. I believe it's nicer this way, i.e. letting the program to exist gracefully and just print a custom message with details about the error.